src/data/ACTIONS/root/GNB.ts

```diff
@@ -257,6 +301,16 @@ export const GNB = ensureActions({
 		cooldown: 120000,
 		statusesApplied: ['NEBULA'],
 	},
+	GREAT_NEBULA: {
+		id: 0, //TODO: Add ID
+		name: 'Great Nebula',
+		icon: 'icon', //TODO: Add icon
+		onGcd: false,
+		cooldown: 120000,
+		statusesApplied: [
+			'GREAT_NEBULA',
```

Comment:
_Nit:_ For consistency with all the other `statusesApplied` properties in this file, might be nice to put this on one line as well instead of spreading it out.

Reply:
Formatting Fixed.
I originally thought the additional affect was a seperate buff. RIP "Not Thrill of Battle",